src/components/SQFormTransferTool/DialogInner.tsx

```diff
@@ -140,7 +140,7 @@ function DialogInner({
       onClose={noop}
     >
       <Form>
-        <DialogTitle sx={classes.title}>
+        <DialogTitle sx={classes.title} component="div">
```

Comment:
I know for `Box` div is the default component. Is that not the case for DialogTitle or is it helping the styling?

Reply:
Your intuition is correct div is not the default component for DialogTitle an H2 is (iirc)
This solved a console error that was something like "an H5 cannot be the child of an H2"